src/node.cc

```diff
@@ -2626,14 +2626,8 @@ void SetupProcessObject(Environment* env,
   READONLY_PROPERTY(process, "arch", OneByteString(env->isolate(), NODE_ARCH));
 
   // process.platform
-#ifdef _WIN32
-  // As determined by gyp, NODE_PLATFORM equals 'win' on windows. However
-  // for historic reasons process.platform should be 'win32'.
-  Local<String> platform = OneByteString(env->isolate(), "win32");
-#else
-  Local<String> platform = OneByteString(env->isolate(), NODE_PLATFORM);
-#endif
-  READONLY_PROPERTY(process, "platform", platform);
+  READONLY_PROPERTY(process, "platform",
+                                 OneByteString(env->isolate(), NODE_PLATFORM));
```

Comment:
The former is the preferred style.  The latter is for when the former would exceed 80 columns.